pandas/plotting/_core.py

```diff
@@ -1470,6 +1471,9 @@ def kde(
             'scott', 'silverman', a scalar constant or a callable.
             If None (default), 'scott' is used.
             See :class:`scipy.stats.gaussian_kde` for more information.
+        weights : NumPy array, optional
+            Weights of datapoints. This must be the same shape as dataset.
```

Comment:
`dataset` is a parameter of `scipy.stats.gaussian_kde`, but it is not explicitly used here. Perhaps `datapoints` is more clear.
```suggestion
            Weights of datapoints. This must be the same shape as datapoints.
```